app/display/editor/doc/access_pv_in_script.rst

```diff
@@ -27,24 +27,32 @@ than one trigger PV for a script and you need to know this execution is triggere
 
 **Examples**:
 
-*Get double value from PV:*
+**Get double value from PV:**
 
 .. code-block:: javascript
```

Comment:
Regarding JavaScript vs. Jython for scripting, see 
https://github.com/ControlSystemStudio/phoebus/wiki/Display-Builder-Script-Compatibility#jython-vs-javascript

Reply:
Copy-paste error. I'll update.

Reply:
On second thoughts.... the syntax `importPackage(Packages.org.csstudio.opibuilder.scriptUtil);` works just fine.

Reply:
While the long-term future of either scripting support in Java is not known, Jython is for now preferred because it has been longer-lived than the changing JS script support. On the wiki I thus only show examples for Jython.

Reply:
I see. Are you asking me to change to Jython?

Reply:
On the wiki I use Jython because it has been the same since the beginning of BOY, while JavaScript support used different libs, required different import statements over time. At the same time, I cannot guarantee that Jython will remain available, and Javascript starts up faster.